lib/src/model.dart

```diff
@@ -6640,14 +6605,16 @@ class PackageBuilder {
     }
   }
 
+  int counter = 0;
+  int counter2 = 0;
```

Comment:
I don't see any reference to these fields. Are they needed?

If so, consider renaming them to indicate what they're counting.

Reply:
whoops, these are debugging leftovers.  Fixed.